spec/support/module/omniauth_macros.rb

```diff
@@ -0,0 +1,24 @@
+module OmniauthMacros
+  def stub_omniauth
```

Comment:
This is awesome!

Reply:
Just realized that we never actually used this lol, switched the test over to this method

Reply:
lmao I noticed the method but not that we didn't use it 🤦 💀